src/components/pages/Classroom/index.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import InstructorSidebar from '../InstructorHome/InstructorSidebar';
+import '../../../styles/ClassroomStyles/index.less';
+import { Card, Button, Layout, Typography, Row, Badge } from 'antd';
+import StudentCard from './StudentCard';
+const { Content } = Layout;
+const { Title } = Typography;
+
+const students = [
+  {
+    id: 'a24efb',
+    name: 'Margery',
+    image:
+      'https://images.unsplash.com/photo-1517841905240-472988babdf9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
+  },
+  {
+    id: 'a24efb',
```

Comment:
Love the dummy data. It might look a little nicer presented if you make the data dynamic and versatile. The avatars on that page looked so good so getting a dozen or so different sets of data would be awesome.